modules/rkhunter/manifests/monitoring.pp

```diff
@@ -44,11 +44,19 @@
   }
 
   # Script to run rkhunter as a passive check
-  file { '/etc/cron.daily/rkhunter-passive-check':
+  file { '/usr/local/bin/rkhunter-passive-check':
     ensure  => present,
     mode    => '0755',
     content => template('rkhunter/rkhunter-passive-check.erb'),
   }
+
+  # Run it twice a day to stop superfluous alerting
+  cron::crondotdee { 'rkhunter-passive-check':
+    command => '/usr/local/bin/rkhunter-passive-check',
+    hour    => '6, 14',
+    mailto  => '',
```

Comment:
Should the default `mailto` be undef or an empty string? It seems odd to use both when i _think_ you just want an empty `MAILTO=`.